### DISCUSSION TITLE
About tutorial

### DISCUSSION THREAD

**Discussion by amrit-thapa-2044** [isce-framework/dolphin#56] [Q&A] [answered]
Is there a tutorial for general users? A Jupyter notebook would be very useful for beginners to learn about this package.

**Comment by scottstanie** [comment]
Hi Armit,
Since the project is pretty new, we've been actively developing the basic capabilities so far. The only walkthrough right now is a work in progress (see PR #26 , https://github.com/opera-adt/dolphin/blob/567ebedbb1c57f6ab327d9036540c32f9900ac2b/docs/notebooks/walkthrough-basic.ipynb) so it only goes into a little more detail than https://dolphin-insar.readthedocs.io/en/latest/getting-started/ .

**Reply by amrit-thapa-2044** [reply]
I appreciate you sharing the walkthrough. I'll begin to explore it shortly.

**Reply by scottstanie** [reply, marked as answer]
This was updated in the last PR and is on the docs now https://dolphin-insar.readthedocs.io/en/latest/notebooks/walkthrough-basic/

**Reply by scottstanie** [reply]
if you have other ideas for what would be useful as tutorial/ How-To guides, please post them too!

**Reply by amrit-thapa-2044** [reply]
This is fantastic! Are there any tutorials like this one for COMPASS? Can we use a stack of SLCs generated using ASF VERTEX [TIF format] with Dolphin?

**Reply by scottstanie** [reply]
Sorry, there aren't any tutorials like this yet for COMPASS or sweets, probably coming in the future.

I haven't used the TI format yet- is it gdal readable? If they are coregistered and can be loaded using `gdal.Open(filename)` or `rasterio.open( ... )` then they should be usable

**Reply by amrit-thapa-2044** [reply]
My bad. It was a typo. It should be ".tif."

**Reply by scottstanie** [reply]
Yep that should be fine, feel free to post if it fails to load them for some reason

**Comment by scottstanie** [comment]
Moving this to a discussion in case you do try it out and have further questions/comments. Please post them, since I'm sure other people will have the same questions, and i'll add them into the walkthrough
